File: templates/Uwp/_comp/MVVMLight/Feat.IdentityForcedLogin/ViewModels/LogInViewModel_postaction.cs
//{[{
using GalaSoft.MvvmLight;
using GalaSoft.MvvmLight.Command;
//}]}

namespace Param_RootNamespace.ViewModels
{
    public class LogInViewModel : ViewModelBase
    {
        public bool IsBusy
        {
            get => _isBusy;
            set
            {
                Set(ref _isBusy, value);
//{[{
                LoginCommand.RaiseCanExecuteChanged();
//}]}
            }
        }
    }
}

File: templates/Uwp/_comp/MVVMLight/Feat.IdentityForcedLogin._VB/ViewModels/LogInViewModel_postaction.vb
'{[{
Imports GalaSoft.MvvmLight
Imports GalaSoft.MvvmLight.Command
'}]}
Namespace ViewModels
    Public Class LogInViewModel
        Inherits ViewModelBase

        Public Property IsBusy As Boolean
            Get
                Return _isBusy
            End Get
            Set(value As Boolean)
                [Set](_isBusy, value)
'{[{
                LoginCommand.RaiseCanExecuteChanged()
'}]}
            End Set
        End Property
    End Class
End Namespace
